src/styles/utils/variables.scss

```diff
@@ -0,0 +1,3 @@
+$day-width: 100px;
+$day-gap: 1px;
+$calendar-width: ($day-width * 7) + ($day-gap * 6);
```

Comment:
You got 6 columns, but there should be 7. You missed the side paddings in the calculations

Reply:
Wow! I was so overwhelmed with these loops and variables, that didn't notice such an obvious mistake. Fixed, thanks!